protocols/ipfs.ts

```diff
@@ -109,8 +109,10 @@ export class IPFSProtocol implements Protocol<Static<typeof IPFSProtocolFields>>
     }
 
     // Default libp2p config: https://github.com/ipfs/helia/blob/main/packages/helia/src/utils/libp2p-defaults.ts
+    const defaults = await libp2pDefaults()
+    
     const libp2pOptions = {
-      ...libp2pDefaults(),
+      ...defaults,
       addresses: {
```

Comment:
@2color We have a public IP, do we need to announce it like this in `addresses` for global visibility?

```json
  announce: [
    `/ip4/<DP_PUBLIC_IP>/tcp/4001`,
    `/ip4/<DP_PUBLIC_IP>/tcp/4001/ws`
```